pyaedt/filtersolutions_core/attributes.py

```diff
@@ -0,0 +1,1528 @@
+        status = self._dll.setGaussianBesselReflection(column.value)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def even_order(self) -> bool:
+        """Filter even order mode status for filter with even orders.
+        The default is `True`.
+
+        Returns
+        -------
+        bool
+        """
+        even_order = c_bool()
+        status = self._dll.getEvenOrderMode(byref(even_order))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(even_order.value)
+
+    @even_order.setter
+    def even_order(self, even_order: bool):
+        status = self._dll.setEvenOrderMode(even_order)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def even_order_refl_zero(self) -> bool:
+        """Filter even order reflection zeros to 0 status.
+        The default is `True`.
+
+        Returns
+        -------
+        bool
+        """
+        even_order_refl_zero = c_bool()
+        status = self._dll.getEvenReflZeroTo0(byref(even_order_refl_zero))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(even_order_refl_zero.value)
+
+    @even_order_refl_zero.setter
+    def even_order_refl_zero(self, even_order_refl_zero: bool):
+        status = self._dll.setEvenReflZeroTo0(even_order_refl_zero)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def even_order_trn_zero(self) -> bool:
+        """Filter even order reflection zeros to infinite status.
+        The default is `True`.
+
+        Returns
+        -------
+        bool
+        """
+        even_order_trn_zero = c_bool()
+        status = self._dll.getEvenTrnZeroToInf(byref(even_order_trn_zero))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(even_order_trn_zero.value)
+
+    @even_order_trn_zero.setter
+    def even_order_trn_zero(self, even_order_trn_zero: bool):
+        status = self._dll.setEvenTrnZeroToInf(even_order_trn_zero)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def constrict_ripple(self) -> bool:
+        """Filter equiripple constriction status.
+        The default is `False`.
+
+        Returns
+        -------
+        bool
+        """
+        constrict_ripple = c_bool()
+        status = self._dll.getConstrictRipple(byref(constrict_ripple))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(constrict_ripple.value)
+
+    @constrict_ripple.setter
+    def constrict_ripple(self, constrict_ripple: bool):
+        status = self._dll.setConstrictRipple(constrict_ripple)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def single_point_ripple(self) -> bool:
+        """Filter ripple confinement to a single frequency point status.
```

Comment:
Same confusion here about interpreting the Boolean. I don't know how many more of these there are, but I am going to quite flagging them!